lib/microservices_router.rb

```diff
@@ -9,7 +9,11 @@ def notify_services_of_changes
                                       Service.all.to_json(include: :models))
       if request.nil?
         Net::HTTP.post_form(URI(s.url),
-                            'error' => 'an error occurred with the request')
+                            'error' => 'Request is nil')
+      elsif request == Net::HTTPSuccess
+      else
+        service_change_error(status)
```

Comment:
It's not clear in context from the method name that this is sending a notification.